src/Protocol/Protocol.h

```diff
@@ -56,9 +56,84 @@ class cProtocol
 
 	virtual ~cProtocol() {}
 
+	/** A list of all outgoing packets */
+	enum eOutgoingPackets
```

Comment:
Would you consider using scoped enums? Or too verbose?

Reply:
I think that'd be pretty verbose, already some lines are going past 80 columns.

Reply:
Don't let yourself be shackled by the width of some IBM punchcard! Believe in the power of the horizontal scrollbar and set yourself free! (or widescreen monitor if your prefer)

Reply:
...but then what should I do with all these boxes of punch cards? And here I thought I was so clever buying them in bulk.

Reply:
hm.
..
..
...I have it! Tape them together, and get a [music box](https://www.youtube.com/watch?v=L4bfyzQsbeM)!

It'll be like programming, but wait, it's musical programming, but *wait*, on punch cards which were designed for actual programming, but *wait*! you \*are* a programmer! Just how deep can we go with the levels of indirection‽